index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header class="Header">
+      <nav>
+        <h1>Elie Harfouche</h1>
+        <div class="lines">
+          <div class="line1"></div>
+          <div class="line2"></div>
+          <div class="line3"></div>
+        </div>
+      </nav>
+    </header>
+    <main>
+      <p class="Geading">Hey There. I’m Elie</p>
+      <p class="dev">I’am a Software Developer</p>
+      <p class="info">
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </p>
```

Comment:
- [ ] Good implementation of the paragraph but I think you can put all in one `<p>` tag, so to go to the next line you can use the `<br/>` tag :upside_down_face: